packages/wallet/.eslintrc.js

```diff
@@ -0,0 +1,241 @@
+const rules = {
```

Comment:
You should edit this file to have the exact same settings as the existing `tslint.json` file so that when you run `yarn lint:write` no changes are made to any files. Then, we can begin toggling linting _settings_ in separate PRs instead of editing the settings in the same PR as introducing a new linting tool.

Reply:
@snario This is quite hard as ESLint and TSLint work differently and have different rules.
I have been trying to get all packages to use the same or similar configs but there will be changes...

Reply:
What do you mean they work differently? Are there certain code styles that `tslint` can check for that `eslint` cannot?

Reply:
The agreed approach now is to use empty rules in this PR, and tackle reintroduction of rules in a future PR.